code/modules/telescience/telescope/quantumTelescope.dm

```diff
@@ -22,7 +22,8 @@ TODO: Enforce ping rate limit here as well in case someone futzes with the javas
 			rebuildEventList(using)
 
 	proc/boot_if_away()
-		if(using && (!using.client || using.client.inactivity >= 600 || !in_interact_range(src, using)))
+		if(using && (!using.client || using.client.inactivity >= 600 || !in_interact_range(src, using) || !isAI(using) && !(GET_DIST(src, using) <= ((WIDE_TILE_WIDTH - 1)/ 2)))) // copied last bit from default.dm tgui code
```

Comment:
Should that new logic be considered a new case to be explicit regarding intent instead of relying on order of operations?

Reply:
It could be if you think it'd be better? I just tacked it on after ``is_interact_range`` since it made sense to check it after that.

Reply:
I just mean warping `!isAI(using) && !(GET_DIST(src, using) <= ((WIDE_TILE_WIDTH - 1)/ 2)))` in parenthesis.